src/OrchardCore.Modules/OrchardCore.Search.Elastic/Services/ElasticIndexManager.cs

```diff
@@ -148,11 +148,17 @@ public async Task<ElasticTopDocs> SearchAsync(string indexName, string query)
             entries.Add("ContentItemId", documentIndex.ContentItemId);
             entries.Add("Id", documentIndex.ContentItemId);
 
+          
+
             foreach (var entry in documentIndex.Entries)
             {
-                if (entry.Name.Contains("Content.ContentItem.FullText") || entry.Name.Contains("Analyzed") || entry.Name.Contains("Sanitize") || entry.Name.Contains("Normalized"))
+                if (entries.ContainsKey(entry.Name)
+                    || entry.Name.Contains("Content.ContentItem.FullText")
```

Comment:
Use defined constant from IndexingConstants.cs so that it be easier to manage.

https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore/OrchardCore.Contents.Core/Indexing/IndexingConstants.cs#L19